usage/managing_documents/uploading_documents.rst

```diff
@@ -21,15 +21,40 @@ The *Document Upload* page looks like the one shown in the picture below.
 
       *Document Upload page*
 
-In order to upload a document:
+On *GeoNode* documents can be:
 
+* Upload from the **Local file**
+
+* Created with reference to **External URL** 
+
+
+In order to upload a document from the **Local file**, you need to:
+
+#. Click on :guilabel:`Select files` button
 #. Select a file from your disk.
 #. Click the :guilabel:`Upload` button
 
+  .. figure:: img/select_files.png
+      :align: center
+
+      *Upload document from the select files option*
+
+For add a document from the **External URL**, you need to:
```

Comment:
I would replace this sentence with the following:

```
| A document may refer to a remote document, without making a local copy of the remote resource.
| To add a document that references an **External URL** you need to:
```

Notice that the pipe chars are there for a reason. They allow splitting the text with new lines without introducing a new paragraph.